src/components/Event.js

```diff
@@ -61,6 +61,7 @@ const Event = props => {
     }
   }
 
+  //add code to render to page here?
```

Comment:
what is this comment referring to?

Reply:
That was a comment for my brain. My bad!

Reply:
I'll remove it